Simplenote/Classes/SearchDisplayController.swift

```diff
@@ -79,6 +79,11 @@ class SearchDisplayController: NSObject {
     func hideNavigationBarIfNecessary() {
         updateNavigationBar(hidden: active)
     }
+
+    func setEnabled(_ enabled: Bool) {
+        searchBar.isUserInteractionEnabled = enabled
+        searchBar.alpha = enabled ? UIKitConstants.alpha1_0 : UIKitConstants.alpha0_5
+    }
```

Comment:
I think this is good 👍 

We may also want to disable search activation via shortcut when in edit mode, right now it is possible.

Reply:
Good suggestion.  Disabled